WinUIGallery/ControlPages/TitleBarPage.xaml

```diff
@@ -129,5 +132,23 @@
                 </StackPanel>
             </local:ControlExample.Example>
         </local:ControlExample>
+
+        <local:ControlExample HeaderText="System titlebar customization"
+                              CSharpSource="Window\TitleBar\TitleBarSample3_cs.txt">
+            <local:ControlExample.Example>
+                <TextBlock TextWrapping="WrapWholeWords">
+                    The OverlappedPresenter is the default presenter for AppWindow, and has more options for customizing the system provided title bar.
+                    Set the title bar to system default to enable these options.
+                </TextBlock>
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <CheckBox Content="Is Maximizable" IsChecked="{x:Bind Presenter.IsMaximizable, Mode=TwoWay}" IsEnabled="{x:Bind IsDefaultTitleBar, Mode=OneWay}"/>
+                    <CheckBox Content="Is Minimizable" IsChecked="{x:Bind Presenter.IsMinimizable, Mode=TwoWay}" IsEnabled="{x:Bind IsDefaultTitleBar, Mode=OneWay}"/>
+                    <CheckBox Content="Has border" IsChecked="{x:Bind HasBorder,Mode=TwoWay}" IsEnabled="{x:Bind HasTitleBar, Converter={StaticResource NegationConverter}, Mode=OneWay}"/>
+                    <CheckBox Content="Has titlebar" IsChecked="{x:Bind HasTitleBar, Mode=TwoWay}" IsEnabled="{x:Bind IsDefaultTitleBar, Mode=OneWay}"/>
```

Comment:
Not sure why we should include this sample since this just makes the app look faulty if set. @gregwoo-microsoft, what do you think?